src/modules/ordersTable/pure/ReceiptModal/index.tsx

```diff
@@ -144,14 +172,27 @@ export function ReceiptModal({
               <OrderTypeField order={order} />
             </styledEl.Field>
 
-            <styledEl.Field>
-              {order.executionData.activityId && (
-                <>
-                  <FieldLabel label={order.executionData.activityTitle} />
-                  <IdField id={order.executionData.activityId} chainId={chainId} />
-                </>
-              )}
-            </styledEl.Field>
+            {/*TODO: add a link to explorer when it will support TWAP orders*/}
+            {!twapOrder && (
+              <styledEl.Field>
+                {order.executionData.activityId && (
+                  <>
+                    <FieldLabel label={order.executionData.activityTitle} />
+                    <IdField id={order.executionData.activityId} chainId={chainId} />
+                  </>
+                )}
+              </styledEl.Field>
+            )}
+            {twapOrder && safeTxParams && (
```

Comment:
Can this be based only on `safeTxParams` instead?
This still requires twapOrder, but should be usable by SWAP and LIMIT orders as well